turbinia/api/utils.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Turbinia API Server helper methods."""
+
+import logging
+import mmap
+import shutil
+import tempfile
+import os
+
+from fastapi import HTTPException
+from turbinia import config as turbinia_config
+
+log = logging.getLogger('turbinia:api_server:utils')
+
+
+def create_zip(request_id: str, task_id: str):
+  """Compress a Turbinia request or task's output directories
+  into a zip file.
+
+  Args:
+    request_id (str): Turbinia request identifier.
+    task_id (str): Turbinia task identifier.
+
+  Returns:
+    A zip compressed bytes stream
+
+  Raises:
+    HTTPException if the request/task output paths could not be found
+        on the file system.
+  """
+  log_path = turbinia_config.toJSON().get('OUTPUT_DIR')
+
+  request_output_path = '{}/{}'.format(log_path, request_id)
```

Comment:
I think we can probably use `os.path.join()` for this.

Reply:
Done